modules/end-user-guide/examples/che-ref_a-kubernetes-secret-as-a-file.adoc

```diff
@@ -68,9 +67,10 @@ data:
   settings.xml: __<base64 encoded data content here>__
 ----
 +
-This results in a file named `settings.xml` being mounted at the `{prod-home}/.m2/` path of all workspace containers.
-+
-The secret-s mount path can be overridden for specific components of the workspace using devfile. To change mount path, an additional volume should be declared in a component of the devfile, with name matching overridden secret name, and desired mount path.
+NOTE: The `data` of the {orch-name} secret may contain several items, whose names must match the desired file name mounted into the container.
```

Comment:
```suggestion
NOTE: The `data` of the {orch-name} secret can contain several items. The names of these items must match the desired file name mounted into the container.
```

Reply:
Or something like `NOTE: The {orch-name} secret `data` element names must match the desired file name mounted into the container.`